src/server/components/formRunner/lawyers.ts

```diff
@@ -15,11 +15,11 @@ const FormRunnerFields: { [key: string]: string } = {
   speakEnglish: "jsonData.speakEnglish",
   contactName: "jsonData.contactName",
   organisationName: "jsonData.organisationName",
-  "address.firstLine": "address.firstLine",
-  "address.secondLine": "address.secondLine",
-  city: "address.city",
-  postCode: "address.postCode",
-  addressCountry: "address.country.name",
+  "address.firstLine": "jsonData.['address.firstLine']",
+  "address.secondLine": "jsonData.['address.secondLine']",
+  city: "jsonData.city",
+  postCode: "jsonData.postCode",
+  addressCountry: "jsonData.addressCountry",
```

Comment:
Do you remember why you made this change? 

The reason why we use `address` and not jsonData.address.firstLine is because it's where we store the canonical address. `address` is connected to country and geolocation. we use this so we can determine the distance of a user to a provider when they're using find service.

There are controls to essentially ignore any changes to address.country (a user cannot change the country they applied to).

Reply:
Yep, I don't know if you remembered but whenever I was demoing this I was getting errors because the data wasn't being populated correctly. When I changed it to this that error didn't come back. I'll go through the code again to job my memory but for some reason this is how the data was formatted 🤷🏿

Reply:
I see the issue. `findListItemByReference` only returns listItem - and not the related `Address` entry. 

`findListItemById` prisma call looks like this
```
prisma.listItem.findUnique({
      where: { id: Number(id) },
      include: {
        address: {
          select: {
            id: true,
            firstLine: true,
            secondLine: true,
            city: true,
            postCode: true,
            country: {
              select: {
                id: true,
                name: true,
              },
            },
          },
        },
        history: {
          orderBy: {
            time: "desc",
          },
        },
        pinnedBy: true,
      },
    });
``` 

and returns an object that looks like 

```
const listItem = {
  jsonData: {..},
  address: { 
    city: '..',
    postCode: '..',
  }
}
```


`findListItemByReference` only returns 

```
const listItem = {
  jsonData: {..}
}
```